hokusai/staging.yml

```diff
@@ -48,10 +48,21 @@ metadata:
     app: hokusai-demo
   name: hokusai-demo
   namespace: default
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:iam::585031190124:server-certificate/2016-01-19_artsy-net-wildcard"
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
```

Comment:
This is the key annotation for getting Kubernetes to create a load balancer with SSL only enabled on the https listener